Content.Client/Audio/ClientAdminSoundSystem.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Content.Shared.Audio;
+using Content.Shared.CCVar;
+using Robust.Shared.Audio;
+using Robust.Shared.Configuration;
+using Robust.Shared.Player;
+
+namespace Content.Client.Audio;
+
+public sealed class ClientAdminSoundSystem : SharedAdminSoundSystem
+{
+    [Dependency] private readonly IConfigurationManager _cfg = default!;
+
+    private List<IPlayingAudioStream?> _adminAudio = new(1);
+
+    public override void Initialize()
+    {
+        base.Initialize();
+        SubscribeNetworkEvent<AdminSoundEvent>(PlayAdminSound);
+        _cfg.OnValueChanged(CCVars.AdminSoundsEnabled, ToggleAdminSound, true);
+    }
+
+    public override void Shutdown()
+    {
+        base.Shutdown();
+        _adminAudio.Clear();
+    }
```

Comment:
You should shutdown the streams as this is called when the sim shuts down and not necessarily the game (i.e. if you try to reconnect).